cellfinder/core/main.py

```diff
@@ -42,6 +42,7 @@ def main(
     detect_callback: Optional[Callable[[int], None]] = None,
     classify_callback: Optional[Callable[[int], None]] = None,
     detect_finished_callback: Optional[Callable[[list], None]] = None,
+    detection_torch_device: Optional[str] = None,
```

Comment:
I think we just want one `torch_device` variable for both detection and classification - I don't think there is a use case where we would run detection on GPU and classification on CPU, or vice versa.